pages/ksphere/kommander/1.1/clusters/attach-eks-cluster/index.md

```diff
@@ -0,0 +1,143 @@
+---
+layout: layout.pug
+navigationTitle: Attach Amazon EKS Cluster to Kommander
+title: Attach Amazon EKS Cluster to Kommander
+menuWeight: 7
+excerpt: Attach an existing EKS cluster to Kommander
+beta: false
+enterprise: false
+---
+
+You can attach existing Kubernetes clusters to Kommander. After attaching the cluster, you can use Kommander to [examine and manage](/ksphere/kommander/1.1/clusters/) this cluster. The following procedure shows how to attach an existing Amazon Elastic Kubernetes Service (EKS) cluster to Kommander.
+
+## Before you begin
+
+This procedure requires the following items and configurations:
+
+- A fully configured and running Amazon [EKS](https://aws.amazon.com/eks/) cluster with administrative privileges.
+- Konvoy v1.5.0 or above, [installed and configured](/ksphere/konvoy/1.5/install/) for your Amazon EKS cluster, on your machine.
+- Kommander v1.0.1-beta.1 or above, [installed and configured](/ksphere/kommander/1.1/install/) on your machine.
```

Comment:
This is a newly added page in 1.1. So is it correct that you could use a v1.0.1 Kommander here? the link points to a 1.1 page

Reply:
seems to be a typo, it should be `1.1.0-beta.1` - again, not changed in this PR tho.